src/controller/ApplicationController.tsx

```diff
@@ -6,7 +6,7 @@ import React, {
   useState,
 } from 'react';
 import type { ComponentType, FC, ReactNode } from 'react';
-// @ts-ignore
+// @ts-ignore TODO update module name in aha-app fork
```

Comment:
I'm unclear what this means -- is this something we should do in this PR?

Reply:
the problem is the types in our fork of react-easy-state still reference the original name: https://github.com/aha-app/react-easy-state/blob/aha-release/types/index.d.ts#L1 - hence the ts-ignore here, which then means `store` is typed `any` in this file.

I'll remove this comment though, it was meant to be more of a note-to-self